### PR TITLE
Neutral background for modals

### PR DESCRIPTION
**Pull request by mfrachet**
Screenshot 2020-08-13 at 10 41 15

### CODE REVIEW

**Comment by Elanhant** [comment]
Is there any particular case where we want this? You can DM in Slack if there's a Figma link or something 🙂

**Comment by Elanhant** [comment]
Never mind, I think I know what this is for. Will comment in Figma.

**Comment by Elanhant** [comment]
Actually no, no need to comment in Figma (at least for now). I think what you need can be achieved by just passing a custom element as `children` to `<StyledModalHeader>`. You can do all the styling in that custom element.

**Comment by mfrachet** [comment]
Yes but it will break the semantic title I think nope?

**Comment by Elanhant** [comment]
> Yes but it will break the semantic title I think nope?

Not if you use a `<span>` or a `<div>` 🙂 
```jsx
<StyledModalHeader>
  <span css={textCss}>
    <MdVisibility css={iconCss} />
    New feature announcement
  </span>
</StyledModalHeader>
```

**Comment by mfrachet** [comment]
Yeah but the header is already a heading: https://github.com/gatsby-inc/gatsby-interface/blob/main/src/components/Modal/StyledModal.tsx#L58 and heading by default is a H2 https://github.com/gatsby-inc/gatsby-interface/blob/main/src/components/Heading/Heading.tsx#L39 ?

**Comment by Elanhant** [comment]
> Yeah but the header is already a heading

Are you planning to put the actual feature title (e.g. "Introducing Gatsby Cloud Workspaces") into `StyledModalHeader`? No need to do that, it should be a part of the modal body. Think of it like this:
* "New feature announcement" is the _modal header_, it describes what the entire modal is about
* Feature title is, well, _the feature title_ — it describes what is the actual feature; it is semantically coupled with the actual content of the modal

**Comment by mfrachet** [comment]
Oh I see! We can approach it that way 😊 Thank you! Tomorrow, I'll revert these changes to only keep the modal overlay in the background! Thanks for your help

File: src/components/Modal/StyledModal.stories.tsx [hunk 1]
```diff
@@ -31,14 +31,20 @@ export default {
...
   decorators: [
-    disableAnimationsDecorator,
     story => (
       <React.Fragment>
         <Global
           styles={(theme: Theme) => [
             {
               body: { background: theme.colors.grey[20] },
             },
+            isChromatic() && {
```
Comment (Elanhant) [inline src/components/Modal/StyledModal.stories.tsx:41]: `disableAnimationsDecorator` is doing exactly the same thing, why remove it? 🙂

Comment (mfrachet) [reply src/components/Modal/StyledModal.stories.tsx:41]: Mmh, this have been introduced by the rebase I think 🤔 . I didn't moved these part of the file. Going to re-rebase on master

Comment (mfrachet) [reply src/components/Modal/StyledModal.stories.tsx:41]: oooh, rebasing master, not main, I forgot about that 😓

**Pull request merged**